grafana-plugin/src/containers/MobileAppVerification/MobileAppVerification.tsx

```diff
@@ -116,13 +127,14 @@ const MobileAppVerification = observer(({ userPk }: Props) => {
           Sign In
         </Text>
         <Text type="primary">Open Grafana IRM mobile application and scan this code to sync it with your account.</Text>
-        <div className="u-width-100 u-flex u-flex-center">
-          <QRCode value={QRCodeValue} />
+        <div className={cx('u-width-100', 'u-flex', 'u-flex-center', 'u-position-relative')}>
+          <QRCode className={cx({ blurry: isQRBlurry })} value={QRCodeValue} />
+          {isQRBlurry && (
```

Comment:
what is the blur intended to do?

locally I just get a very fast flicker. only if I throttle my Chrome's network activity to "Slow 3G" do I see the blurred/loading state 🤔